src/Mollie/WC/Helper/ApplePayDirectHandler.php

```diff
@@ -0,0 +1,1180 @@
+<?php
+
+class Mollie_WC_Helper_ApplePayDirectHandler
+{
+    const VALIDATION = 'mollie_apple_pay_validation';
+    const CREATE_ORDER = 'mollie_apple_pay_create_order';
+    const CREATE_ORDER_CART = 'mollie_apple_pay_create_order_cart';
+    const UPDATE_SHIPPING_CONTACT = 'mollie_apple_pay_update_shipping_contact';
+    const UPDATE_SHIPPING_METHOD = 'mollie_apple_pay_update_shipping_method';
+    const REDIRECT = 'mollie_apple_pay_redirect';
+
+    /**
+     * Initial method that checks if the device is compatible
+     * if so puts the button in place
+     * and adds all the necessary actions
+     */
+    public function bootstrap()
+    {
+        if (!$this->isApplePayCompatible()) {
+            add_action(
+                'admin_notices',
+                function () {
+                    echo '<div class="error"><p>';
+                    echo sprintf(
+                        esc_html__(
+                            '%1$sServer not compliant with Apple requirements%2$s Check %3$sApple Server requirements page%4$s to fix it in order to make the Apple Pay button work',
+                            'mollie-payments-for-woocommerce'
+                        ),
+                        '<strong>',
+                        '</strong>',
+                        '<a href="https://developer.apple.com/documentation/apple_pay_on_the_web/setting_up_your_server">',
+                        '</a>'
+                    );
+                    echo '</p></div>';
+                    return false;
+                }
+            );
+
+            return;
+        }
+
+        if (!$this->merchantValidated()) {
+            add_action(
+                'admin_notices',
+                function () {
+                    echo '<div class="error"><p>';
+                    echo sprintf(
+                        esc_html__(
+                            '%1$sApple Pay Validation Error%2$s Check %3$sApple Server requirements page%4$s to fix it in order to make the Apple Pay button work',
+                            'mollie-payments-for-woocommerce'
+                        ),
+                        '<strong>',
+                        '</strong>',
+                        '<a href="https://developer.apple.com/documentation/apple_pay_on_the_web/setting_up_your_server">',
+                        '</a>'
+                    );
+                    echo '</p></div>';
+                    return false;
+                }
+            );
+        }
```

Comment:
Why not create a method you can reuse instead of long and complex output?

```
private function addAdminNotice($level, $message) 
{
   add_action(
       'admin_notice', 
       function() use ($level, $message) {
          ?>
          <div class="notice notice-<?= esc_attr($level) ?>">
                  <?= wp_kses_post($message) ?>
          </div>
          <?php
       }
    );
}
``` 
Sort of.

Additionally I would split the message into multiple `sprintf` and then mount it, to make it more maintainable and simple to change if needed.